content/millennium/r4/financial/coverage.md

```diff
@@ -196,7 +211,8 @@ _Implementation Notes_
 
 * This implementation follows the [JSON Patch](https://tools.ietf.org/html/rfc6902) spec.
 * Only operations on the paths listed below are supported.
-* Only Encounter-level Coverages may be patched. Patches are not currently supported for Patient-level Coverages.
+* For Private Coverages, only Encounter-level Coverages may be patched. For Public Coverages,
+both Encounter-level and Patient-level Coverages may be patched, with the caveat of only supporting the `/period` and `/class/0/value` operations.
```

Comment:
This doesn't say that "Patches are not currently supported for Patient-level Coverages" anymore. Previously it did.